README.md

```diff
@@ -212,6 +216,40 @@ OPTIONS:
     -h, --help               Print help information
 ```
 
+#### dev
+
+This subcommand starts the Lenra app of the current directory in dev mode.
+
+```bash
+$ lenra dev --help
+lenra-dev 
+Starts the dev mode
+
+USAGE:
+    lenra dev [OPTIONS]
+
+OPTIONS:
+        --config <CONFIG>    The app configuration file [default: lenra.yml]
+    -h, --help               Print help information
+```
+
+#### init
+
+This subcommand generates Docker and Docker Compose files.
+
+```bash
+$ lenra init --help
+lenra-init 
+Generates dockerfile and docker compose file with the init command
```

Comment:
```suggestion
Generates Dockerfile and Docker Compose files
```

Reply:
I just copied the help result. We need to change the comment in the command enum to change it: https://github.com/lenra-io/lenra_cli/blob/cc4bc8dc886d8811b81a3d90fd52c84658a94636/src/cli/mod.rs#L37